contrib/kind-common.sh

```
@@ -1844,6 +1882,20 @@ install_ipsec() {
   fi
 }
 
+# run_script_in_container should be used when the script is run nested in a container
+# and makes sure the control-plane node is reachable by substituting 127.0.0.1
+# with the control-plane container's IP.
+run_script_in_container() {
+  if [ "$PLATFORM_IPV4_SUPPORT" == true ]; then
+    local master_ip=$("$OCI_BIN" inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' "${KIND_CLUSTER_NAME}-control-plane" | head -n 1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)